wallet/chainntfns.go

```diff
@@ -419,10 +552,40 @@ func (w *Wallet) addRelevantTx(rec *wtxmgr.TxRecord,
 			if w.StakeMgr.CheckHashInStore(&txInHash) {
 				w.StakeMgr.InsertSSGen(&block.Hash,
 					int64(block.Height),
-					tx.Sha(),
+					&txHash,
 					w.VoteBits,
 					&txInHash)
 			}
+
+			// If we're running as a stake pool, insert
+			// the stake pool user ticket update too.
+			if w.stakePoolEnabled {
+				txInHeight := tx.MsgTx().TxIn[1].BlockHeight
+				poolTicket := new(wstakemgr.PoolTicket)
+				poolTicket.Ticket = txInHash
+				poolTicket.HeightTicket = txInHeight
+				poolTicket.Status = wstakemgr.TSVoted
+				poolTicket.SpentBy = txHash
+				poolTicket.HeightSpent = uint32(block.Height)
+
+				poolUser, err := w.StakeMgr.SStxAddress(&txInHash)
+				if err != nil {
+					log.Warnf("Failed to fetch stake pool user for "+
+						"ticket %v (voted ticket)", txInHash)
+				} else {
+					err = w.StakeMgr.UpdateStakePoolUserTickets(poolUser,
+						poolTicket)
+					if err != nil {
```

Comment:
No, it's an update to a metadata bucket and shouldn't result in a failure to otherwise process the vote.